lib/menu.rb

```diff
@@ -0,0 +1,14 @@
+class Menu
+
+  attr_reader :food
+
+  def initialize(food)
+    @food = food
+  end
+
+  def list
+    food.map do |dish, price|
```

Comment:
I love the way you have done this, able to change a hash table to a nice formatted friendly list for the people who are running this to look at!